### PR TITLE
709 DELETE and GET by id for owned entities

### LINKED ISSUE #709: [Nox.Generators] DELETE and GET By Id from owned entities in Parent Entity Odata Controller
Add endpoint to Parent controller to delete owned entity using the following signature as example - 
`public async Task<ActionResult> DeleteCountryLocalNamesFromCountry([FromRoute] System.Int64 key, [FromRoute] System.String relatedKey)`

Add endpoint to get Owned Entity by Id  api/Countries/1/CountryLocalNames/001
`public Task<ActionResult<CountryLocalNameDto>> GetCountryLocalNames([FromRoute] System.Int64 key, [FromRoute] System.String relatedKey) `

### PR DESCRIPTION
**Pull request by anna-naumova25**
- Removed [EnableQuery] from Post, Put, Patch
- Added Delete endpoint for owned entities
- Added Get by id endpoint for owned entities

TODO:
Action/Function for non-conventional endpoints

### CODE REVIEW

File: src/Nox.Generator/Application/Commands/DeleteOwnedCommand.template.cs
Comment (rochar) [inline src/Nox.Generator/Application/Commands/DeleteOwnedCommand.template.cs:1]: @emir-sehovic1  @anna-naumova25  and ETag is committed we need also to update this new command

File: src/Nox.ClientApi.Tests/Tests/Controllers/CountriesControllerTests.cs [hunk 1]
```diff
@@ -153,6 +153,33 @@ public async Task Get_OwnedEntityByParentKeyAndFilter_ReturnsSingleEntity()
...
+            // Arrange
+            var dto = new CountryCreateDto
+            {
+                Name = _fixture.Create<string>(),
+                CountryLocalNames = new List<CountryLocalNameCreateDto>() {
+                    new CountryLocalNameCreateDto() { Name = expectedCountryLocalName }
+                }
+            };
+            var result = await _oDataFixture.PostAsync<CountryCreateDto, CountryDto>(EntityUrl, dto);
+            var country = await _oDataFixture.GetAsync<CountryDto>($"{EntityUrl}/{result!.Id}");
```
Comment (rochar) [inline src/Nox.ClientApi.Tests/Tests/Controllers/CountriesControllerTests.cs]: we have new GetOdataSingleResponse for this,  you juts need to update the test

**Review by rochar — APPROVED** [review]
looks good, juts update and fix the tests

File: src/Nox.Generator/Application/Commands/DeleteOwnedCommand.template.cs
Comment (anna-naumova25) [reply src/Nox.Generator/Application/Commands/DeleteOwnedCommand.template.cs:1]: Updated

File: src/Nox.ClientApi.Tests/Tests/Controllers/CountriesControllerTests.cs (hunk 1 above)
Comment (anna-naumova25) [reply src/Nox.ClientApi.Tests/Tests/Controllers/CountriesControllerTests.cs]: Updated. Also I've added Skip for one test that is pending OData Action/Function implementation for non-conventional endpoints.

**Review by rochar — APPROVED** [review]

**Pull request merged**